### ISSUE TITLE
high contrast mode - cursor position invisible when selecting text

### ISSUE THREAD

**Issue by mousetraps** [Microsoft/vscode#17140]
- VSCode Version: Code 1.8.0 (38746938a4ab94f2f57d9e1309c51fd6fb37553d, 2016-12-13T17:45:32.595Z)
- OS Version: Windows_NT ia32 10.0.14965

![image](https://cloud.githubusercontent.com/assets/762848/21159709/0b888f06-c137-11e6-8de5-13b4982a36fd.png)

**Comment by alexandrudima** [comment]
Changed the selection color to a yellowish color and added borders to the cursor in case there is a selection. Hopefully this is good enough to distinguish the cursor from the selection. I've also tried using colors, but I've found coming up with two colors that have a high contrast to black and to each other difficult. fyi @bgashler1 

![image](https://cloud.githubusercontent.com/assets/5047891/22240278/a97a7bd0-e21a-11e6-92aa-f87d6ecdacc4.png)
